frame/democracy/src/benchmarking.rs

```diff
@@ -296,13 +299,14 @@ benchmarks! {
 	// Worst case scenario, we external propose a previously blacklisted proposal
 	external_propose {
 		let p in 1 .. MAX_PROPOSALS;
+		let v in 1 .. MAX_VETOERS;
 
 		let origin = T::ExternalOrigin::successful_origin();
 		let proposal_hash = T::Hashing::hash_of(&p);
 		// Add proposal to blacklist with block number 0
 		Blacklist::<T>::insert(
 			proposal_hash,
-			(T::BlockNumber::zero(), vec![T::AccountId::default()])
+			(T::BlockNumber::zero(), (0..v).map(|_| T::AccountId::default()).collect::<Vec<_>>())
```

Comment:
shouldn't `vec![T::AccountId::default(); v]` work as well?
seems a bit simpler
```suggestion
			(T::BlockNumber::zero(), vec![T::AccountId::default(); v])
```